### ISSUE TITLE
Found both group and host with same name: localhost

### ISSUE THREAD

**Issue by kukuhuihao** [StreisandEffect/streisand#1023]
how to solve this problem when i install straisand in terminal
[Mac os 10.13]

**Comment by nopdotcom** [comment]
This message always shows up for me, when streisand works, or when streisand doesn't work. Try running the `./streisand` again and ignoring this?

**Comment by kukuhuihao** [comment]
thank you

**Comment by nopdotcom** [comment]
I'm going to keep this open as a bug so we remember to either shut up the warnings, or explain "Ignore the following message about 'found group with the same'".

**Comment by cpu** [comment]
I agree that we should clean this up :-) I've been ignoring this warning for a long time and should have made an issue, apologies!

**Comment by kukuhuihao** [comment]
Now i meet a new problem，it shows **"./streisand: is a directory"** when i enter **"./streisand:"** in the terminal. @cpu

**Comment by cpu** [comment]
> Now i meet a new problem，it shows "./streisand: is a directory" when i enter "./streisand:" in the terminal. @cpu

@kukuhuihao You need to be in the `streisand` directory you cloned the project to before running `./streisand`. E.g. try running:

1. `cd streisand`
2. `./streisand`
